frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -149,12 +149,8 @@ export default {
 <style lang="scss" scoped>
 @import "@assets/style/_variables.scss";
 
-.c-addPeople {
```

Comment:
But this class is still being used on the button..?

Reply:
Not anymore (now) 😅